src/engraving/libmscore/input.cpp

```diff
@@ -212,6 +212,8 @@ void InputState::update(Selection& selection)
 
     Element* e = selection.element();
     if (e == 0) {
+        setTrack(selection.activeTrack());
+        setSegment(selection.startSegment());
```

Comment:
ensures that when an empty measure selected the active track/segment are set correctly.
Note that this was broken in MU3 too.